_infra/helm/collection-instrument/values.yaml

```diff
@@ -69,3 +69,6 @@ gcp:
   topic: topic
 
 saveSeftInGcp: false
+
```

Comment:
is there a dev key we can use as the default rather than preprod?

Reply:
we can create one, we would need to store the dev private key and need an email address